expression/column.go

```diff
@@ -476,11 +476,11 @@ func indexCol2Col(cols []*Column, col *model.IndexColumn) *Column {
 	return nil
 }
 
-// IndexInfo2Cols gets the corresponding []*Column of the indexInfo's []*IndexColumn,
+// IndexInfo2PrefixCols gets the corresponding []*Column of the indexInfo's []*IndexColumn,
 // together with a []int containing their lengths.
 // If this index has three IndexColumn that the 1st and 3rd IndexColumn has corresponding *Column,
 // the return value will be only the 1st corresponding *Column and its length.
-func IndexInfo2Cols(cols []*Column, index *model.IndexInfo) ([]*Column, []int) {
+func IndexInfo2PrefixCols(cols []*Column, index *model.IndexInfo) ([]*Column, []int) {
```

Comment:
Can we merge the function and `IndexInfo2Cols` as
```go
func IndexInfo2Cols(cols []*Column, index *model.IndexInfo) (preNoNilCols, retCols []*Column, preLengths, lengths []int)
```

It seems most places, which call the `IndexInfo2PrefixCols`, also call the `IndexInfo2Cols`.

Reply:
Seems a good idea. But the return value is a little much. I'll first try @eurekaka 's comment. Then try to merge this two function.

Reply:
Okay~